librz/core/cmd/cmd_alias.c

```diff
@@ -0,0 +1,184 @@
+// SPDX-FileCopyrightText: 2024 RizinOrg <info@rizin.re>
+// SPDX-License-Identifier: LGPL-3.0-only
+
+#include "rz_cmd.h"
+#include "rz_core.h"
+
+static const char *help_msg_dollar[] = {
+	"Usage:", "$alias[=cmd] [args...]", "Alias commands and strings (see %$? for help on $variables)",
+	"$", "", "list all defined aliases",
+	"$*", "", "list all the aliases as rizin commands in base64",
+	"$**", "", "same as above, but using plain text",
+	"$", "foo:=123", "alias for 'f foo @ 123'",
+	"$", "foo-=4", "alias for 'fm $$-4 @ foo'",
+	"$", "foo+=4", "alias for 'fm $$+4 @ foo'",
+	"$", "foo", "alias for 's foo' (note that command aliases can override flag resolution)",
+	"$", "dis=base64:cGRm", "alias this base64 encoded text to be executed when $dis is called",
+	"$", "dis=$hello world", "alias this text to be printed when $dis is called",
+	"$", "dis=-", "open cfg.editor to set the new value for dis alias",
+	"\"$", "dis=af;pdf\"", "create command - analyze to show function", // Move quotation marks inside
+	"\"$", "test=#!pipe node /tmp/test.js\"", "create command - rlangpipe script", // Move quotation marks inside
+	"$", "dis=", "undefine alias",
+	"$", "dis", "execute the previously defined alias",
+	"$", "dis?", "show commands aliased by $dis",
+	"$", "dis?n", "show commands aliased by $dis, without a new line",
```

Comment:
probably these should be defined in the yaml file.

Reply:
Ok 3abe7b782ba95fd2a3c19d29e06d4c2f64e7e51c